doc/sphinx/tutorial/getting-started-with-mir.md

```diff
@@ -33,123 +36,92 @@ sudo apk add mir-demos mir
-
-We'll work with `miral-app`, a script that handles running a Mir shell with a basic GUI.
-
-Mir allows you to run a graphical shell in *desktop mode* or in *kiosk mode*.
+## Run a Mir compositor nested in Wayland or X11
+Next, we will run one of the demo compositors that we just installed, namely `miral-app`.
+`miral-app` is a simple, standalone demo compositor that provides a floating window manager.
 
-Desktop mode means that application windows that are opened are floating, you can move them around the screen, maximize, or minimize them.
-Kiosk mode means that the application is opened in fullscreen mode. You are unable to move or resize it.
+Within your current graphical session, run:
 
-To run the script in desktop mode:
 ```sh
 miral-app
 ```
 
-To run the script in kiosk mode:
-```sh
-miral-app -kiosk
-```
-
-## Running shells natively
-
-Mir compositors support running "natively" by launching them from a virtual terminal or a login screen.
-
-***Do note that while, like many other Wayland compositors, Mir supports Nvidia
-graphics cards they can cause stability issues due to quirks on the driver side.***
+You should see an X11 window on your desktop. This is a full Wayland compositor
+running nested inside your current session.
 
+## Run a Mir compositor natively
+Now that we've run `miral-app` in a nested session, let's run it natively. There are two ways to do this:
+1. Launch the compositor from a virtual terminal
+2. Launch the compositor from a login screen
 
 ### Launching from a virtual terminal
+Let's begin by launching the compositor from a virtual terminal.
+
+To do this, you can "vt-switch" from your current graphical session to a virtual terminal.
+This is typically done by pressing `CTRL+ALT+F\<Number\>` and logging in.
 
-1. Switch to a virtual terminal by pressing CTRL+ALT+F\<Number\> and log in.
+Once logged in, run:
 
-2. Run the script:
 ```sh
 miral-app
 ```
 
-### Launching from a login screen
-
-Open the window manager list, choose **Mir Shell**, log in, and `miral-shell` will be running fullscreen.
+The compositor should appear across your outputs.
 
-## Using on-screen keyboards
-Mir-based compositors support on-screen keyboards. 
+***Do note that while, like many other Wayland compositors, Mir supports Nvidia
+graphics cards they can cause stability issues due to quirks on the driver side.***
 
-**Note**: Due to security reasons, some Wayland extensions needed by on-screen keyboards are disabled by default. In this tutorial, we override this setting by passing `--add-wayland-extensions all` when launching `miral-app`.
-passing a `--add-wayland-extensions all` flag when launching an example application.
+### Launching from a login screen
+Next, let's launch `miral-app` from the login screen. In either gdm or lightdm,
+you will find a dropdown list that contains the list of available Wayland compositors.
+Open this list and choose **Mir Shell**. Log in, and `miral-shell` will be running.
```

Comment:
We don't install a .desktop to `/usr/share/wayland-sessions` anymore (since about 2.18), so this won't work.